iosHackthonApp/Modules/Features/SignUp/Sources/SignUp/SignUpView.swift

```diff
@@ -13,6 +13,8 @@ import Components
 
 public struct SignUpView: View {
 	
+	private let signUpWithEmailViewModel = SignUpWithEmail(viewModel: SignUpWithEmailViewModel())
```

Comment:
I find this naming a little confusing. What do you think to changing it to `SignUpWithEmailView`?

Reply:
yup sounds good..will do

Reply:
Done